File: 1304/Solution.java
import java.util.ArrayList;
import java.util.List;

class Solution {
    public int[] sumZero(int n) {
        List<Integer> list = new ArrayList<>();

        if (n % 2 != 0) {
            list.add(0);
        }

        for (int i = 1; i <= n/2; i++) {
            list.add(i);
            list.add(-i);
        }

        int[] result = new int[list.size()];
        for (int i = 0; i < list.size(); i++) {
            result[i] = list.get(i);
        }
        return result;
    }
}

File: 1304/Solution.py
class Solution(object):
    def sumZero(self, n):
        lst = []
        if n % 2 != 0:
            lst.append(0)
        
        for i in range(1, n//2 + 1):
            lst.append(i)
            lst.append(-i)
        
        return lst   